summoner-cli/src/Summoner/Default.hs

```diff
@@ -75,95 +72,95 @@ currentYear = do
-    license = "$licenseName"
-
-    # Create projects with the Cabal support
-    # cabal = true
-
-    # Create projects with the Stack support
-    # stack = true
-
-    # Should your projects have the library stanza by default?
-    # lib = true
-
-    # Should your projects have an executable stanza by default?
-    # exe = true
-
-    # Should your projects have the test-suite stanza by default?
-    # test = true
-
-    # Should your projects have the benchmark stanza by default?
-    # bench = true
-
-    # Summoner suports 'git' version control system and GitHub as a platform for
-    # hosting 'git' repos. The following set of flags controls integration with GitHub.
-    # github = true    # enabled GitHub integration
-    # noUpload = true  # Init git repo, but don't create it on GitHub
-    # private = true  # create private repos by default
-    # gitignore =      # list of additional entries to be added in .gitignore
-    #     [ "build"
-    #     , "result"
-    #     ]
-
-    # Summoner supports various CI services. Uncomment those you want to use by default.
-    # githubActions = true  # GitHub Actions CI
-    # travis = true         # Travis CI
-    # appveyor = true       # AppVeyor CI
-
-    # List of additional GHC versions to support besides $defaultGhcVer.
-    # Run the 'summon show ghc' command to see the list of all supported GHC versions.
-    # ghcVersions = ["$defaultGhcPrevPrev", "$defaultGhcPrev"]
-
-    # List of default-extensions in the .cabal file
-    # extensions = [ "ConstraintKinds"
-    #              , "DeriveGeneric"
-    #              , "GeneralizedNewtypeDeriving"
-    #              , "InstanceSigs"
-    #              , "KindSignatures"
-    #              , "LambdaCase"
-    #              , "OverloadedStrings"
-    #              , "RecordWildCards"
-    #              , "ScopedTypeVariables"
-    #              , "StandaloneDeriving"
-    #              , "TupleSections"
-    #              , "TypeApplications"
-    #              , "ViewPatterns"
-    #              ]
-
-    # List of additional files to add after creating the project
-    # files =
-    #     [ { path = ".stylish-haskell.yaml"
-    #       , url  = "https://raw.githubusercontent.com/kowainik/org/master/.stylish-haskell.yaml"
-    #       }
-    #     , { path = ".github/CODEOWNERS"
-    #       , raw  = "*  @$defaultOwner"
-    #       }
-    #     , { path  = "src/Foo.hs"
-    #       , local = "/home/user/.default/Foo.hs"
-    #       }
-    #     ]
-
-    # Alternative Prelude to be used instead of the default one if you prefer
-    # [prelude]
-    #     package = "relude"
-    #     module  = "Relude"
-|]
+defaultConfigFileContent = unlines
+    [ "# This file is automatically generated by Summoner."
+    , "# Edit required fields, uncomment additional settings or delete irrelevant"
+    , "# lines for the personalized configuration."
+    , ""
+    , "# GitHub user name"
+    , "owner = \"" <> defaultOwner <> "\""
```

Comment:
I feel like we will benefit from having the following function in `Summoner.Text`:

```haskell
quote :: Text -> Text
quote t = "\"" <> t <> "\""
```

Not necessary required right now, but probably will be useful in the future, so something to keep in mind.